pandas/tests/indexing/test_datetime.py

```diff
@@ -8,6 +8,14 @@
 
 class TestDatetimeIndex(object):
 
+    def test_setitem_with_datetime_tz(self):
+        mask = np.array([True, False, True, False])
```

Comment:
add a comment with the issue number

can you also add the working example as well (from your issue)